server/neptune/temporalworker/server.go

```diff
@@ -40,25 +40,29 @@ import (
 const (
 	ProjectJobsViewRouteName = "project-jobs-detail"
 
-	// Equal to default terraform timeout
-	TemporalWorkerTimeout = time.Hour
+	// to make this clear,
+	// time t     event
+	// 0 min      sigterm received from kube
+	// 50 min     activity ctx canceled
+	// 50 + x min sigkill received from kube
+	//
+	// Note: x must be configured outside atlantis and is the grace period effectively.
+	TemporalWorkerTimeout = 50 * time.Minute
```

Comment:
`x must be configured outside atlantis`. Is this grace period between sigterm and sigkill different from https://github.com/lyft/atlantis/blob/706ad3a9943c929224985ede6de6a03031cdbbc3/server/neptune/workflows/activities/terraform/async_client.go#L134

Reply:
yes, x is when the outside orchestrator shuts us down (in our case that's kube)